tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/EndpointSchemaGeneratorMojo.java

```diff
@@ -1051,9 +1051,8 @@ protected void findComponentClassProperties(
                             .filter(opt -> name.equals(opt.getName())).findAny();
                     if (prev.isPresent()) {
                         String prv = prev.get().getJavaType();
-                        String cur = fieldTypeName;
```

Comment:
using `cur` for current makes the code easier to read, so lets keep the code as-is